src/testrender/optixraytracer.h

```diff
@@ -32,15 +32,7 @@ class OptixRaytracer final : public SimpleRaytracer {
                              const std::string& var_name)
     {
         ustring ustr = ustring(str);
-#if OPTIX_VERSION < 70000
-        uint64_t addr = m_str_table.addString(ustr, ustring(var_name));
-        if (!var_name.empty())
-            register_global(var_name, addr);
-        return addr;
-#else
-        m_hash_map[ustr.hash()] = ustr.c_str();
```

Comment:
Right, will fix. But also I'm going to eliminate this map (in a future PR) because I recently added a facility to ustring itself that lets you recover the ustring from the hash, so we won't need to keep this map in OSL at all.